packages/cli/package.json

```diff
@@ -50,7 +51,8 @@
     "@blitzjs/server": "0.5.0-canary.1",
     "@oclif/dev-cli": "^1.22.2",
     "@oclif/test": "^1.2.5",
-    "@prisma/cli": "2.0.0-beta.2"
+    "@prisma/cli": "2.0.0-beta.2",
+    "@types/node-fetch": "^2.5.6"
```

Comment:
Can you move this types dependency to the root? All dev dependencies should be installed at root from now on (less weird monorepo issues)